fetch.bs

```diff
@@ -4880,36 +4879,37 @@ initially a new {{AbortSignal}} object.
- <dd>Returns the referrer of <var>request</var>. Its value can be a same-origin URL if explicitly
- set in <var>init</var>, the empty string to indicate no referrer, and "<code>about:client</code>"
- when defaulting to the global's default. This is used during fetching to determine the value of
- the "<code>Referer</code>" header of the request being made.
+ <dd>Returns the referrer of <var>request</var>. Its value can be a same-origin URL if
+ explicitly set in <var>init</var>, the empty string to indicate no referrer, and
+ "<code>about:client</code>" when defaulting to the global's default. This is used during
+ fetching to determine the value of the `<code>Referer</code>` header of the request being made.
 
  <dt><code><var>request</var> . <a attribute for=Request>referrerPolicy</a></code>
  <dd>Returns the referrer policy associated with <var>request</var>. This is used during
  fetching to compute the value of the <var>request</var>'s referrer.
 
  <dt><code><var>request</var> . <a attribute for=Request>mode</a></code>
- <dd>Returns the mode associated with <var>request</var>, which is a string indicating whether
+ <dd>Returns the <a>mode</a> associated with <var>request</var>, which is a string indicating whether
  or not the request will use CORS, or will be restricted to same-origin URLs.
 
  <dt><code><var>request</var> . <a attribute for=Request>credentials</a></code>
- <dd>Returns the credentials mode associated with <var>request</var>, which is a string
+ <dd>Returns the <a>credentials mode</a> associated with <var>request</var>, which is a string
  indicating whether or not credentials will be sent with the request always, never, or only when
  sent to a same-origin URL.
 
  <dt><code><var>request</var> . <a attribute for=Request>cache</a></code>
- <dd>Returns the cache mode associated with <var>request</var>, which is a string indicating
+ <dd>Returns the <a>cache mode</a> associated with <var>request</var>, which is a string indicating
  how the the request will interact with the browser's cache when fetching.
 
  <dt><code><var>request</var> . <a attribute for=Request>redirect</a></code>
- <dd>Returns the redirect mode associated with <var>request</var>, which is a string indicating
- how redirects for the request will be handled during fetching. A <a>request</a> will follow
- redirects by default.
+ <dd>Returns the <a>redirect mode</a> associated with <var>request</var>, which is a string
+ indicating how redirects for the request will be handled during fetching. A <a>request</a>
+ will follow redirects by default.
 
  <dt><code><var>request</var> . <a attribute for=Request>integrity</a></code>
- <dd>Returns <var>request</var>'s subresource integrity, which is a cryptographic hash of the
+ <dd>Returns <var>request</var>'s subresource integrity metadata, which is a cryptographic hash of the
  resource being fetched. Its value may consist of multiple hashes separated by whitespace.
+ [[!SRI]]
```

Comment:
You need to omit the ! as it's a non-normative reference in this case. I don't think it ends up mattering at the moment, but it's good to have it correct in case it ends up mattering some day.